metadata.yaml

```diff
@@ -293,13 +293,30 @@ root:
         is_force_new: false
         is_provision_controller: false
         is_count_controller: false
+      - name: "is_enable_odf"
+        type : bool
+        is_required: false
+        description: "Set to true to enable ODF install"
+        example: true
+        is_sensitive: false
+        related_to: ["null_resource.odf"]
+        is_nullable: false
+        is_force_new: false
+        is_provision_controller: false
+        is_count_controller: false
     outputs:
       - name: "portworx_is_ready"
```

Comment:
portworx needs to removed here?

Reply:
Portworx and odf are combined in this same directory.